brcm_fwpatch_ldr.py

```diff
@@ -14,15 +14,25 @@ def accept_file(li, n):
         pass
     return 0
 
-def make_seg(addr_range):
-    seg = idaapi.segment_t()
-    seg.bitness = 1 # 32bit
-    seg.startEA = addr_range[0]
-    seg.endEA = addr_range[1]
-    seg.perm = idaapi.SEGPERM_READ | idaapi.SEGPERM_WRITE | idaapi.SEGPERM_EXEC
-    seg.type = idaapi.SEG_CODE
-    # use bss to tell what is not patched
-    idaapi.add_segm_ex(seg, 'patchram', 'BSS', idaapi.ADDSEG_NOSREG)
+def make_seg(addr_range, type):
+    if type:
```

Comment:
Looks like duplicate code here and type is a python keyword. Could you change it to another name and remove duplication?